docs/Data.md

```diff
@@ -116,7 +116,7 @@ This format allows to include data directly in the Gosling's JSON specification.
 | chromosomeField    | string                            | specify the name of chromosome data fields                   |
 | genomicFields      | string[]                          | specify the name of genomic data fields                      |
 
-## BigWig
+### BigWig
```

Comment:
FIY, the table of content is automatically generated using vscode. So, if the table of content has some problem (incorrect indent), it means the main content has the same problem

Reply:
Ah, I see. Thank you for the info!